test/documentation/seqan3.css

```diff
@@ -255,3 +255,12 @@ div.solution {
     content: " Class";
     color: grey;
 }
+
+/* Linebreak in template parameters: */
+.memTemplParams {
+    white-space: normal;
+}
+
+.memItemLeft {
+    white-space: normal;
```

Comment:
Can you try how the following looks on your browser:

```css
.memItemLeft {
    display: block ruby;
    max-width: 300px;
    overflow-block: scroll;
}
```
It adds a max width and a scroll bar for the types that are too long for me which maybe is a compromise :)

Reply:
Have you tested this with different browsers. It seems that neither `block ruby` nor `overflow-block` are supported by all browsers.

Reply:
It does not work at all actually :sweat_smile: :see_no_evil: